content/realtime/channels.textile

```diff
@@ -236,6 +236,149 @@ channel.subscribe { message in
 }
 ```
 
+h4(#rewind). Subscribing to a channel with rewind option enabled
+
+If you wish to both subscribe to a channel, as well as obtain some messages from said channel's "history":/realtime/history seamlessly, you can make use of the "rewind":/realtime/channel-params feature. This will result in the specified number of messages from "history":/realtime/history being returned as part of the subscription process.
+
+minimize. View details
+  "Rewind":/realtime/channel-params has two methods of obtaining messages from history, either by getting a specified number of messages from history, or messages from a set period of time into the past.
```

Comment:
I would split these into two clear distinctions, probably with bullet points rather than a single sentence. Or simple break it into two sentences. e.g.

Rewind":/realtime/channel-params has two methods of obtaining messages from history. You can retreive a specified number of messages from history or messages from a set period of time into the past.